resources/scrolling/neutrallScroll.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
misspelled filename
```diff
-neutrallScroll.svg
+neutralScroll.svg

-neutrallScroll.png
+neutralScroll.png
```

Reply:
Oh thanks, I am German and forgot how to spell `neutral`